app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    persons = [Person(pep["name"], pep["age"]) for pep in people]
```

Comment:
Use descriptive names, you can split into lines

Reply:
i`m not sure about corectly understand you comment, i try fix the descriptive names, check please